tests/components/axis/test_init.py

```diff
@@ -10,21 +20,21 @@
 
 async def test_setup_no_config(hass):
     """Test setup without configuration."""
-    assert await async_setup_component(hass, axis.DOMAIN, {})
-    assert axis.DOMAIN not in hass.data
+    assert await async_setup_component(hass, AXIS_DOMAIN, {})
+    assert AXIS_DOMAIN not in hass.data
 
 
 async def test_setup_entry(hass):
     """Test successful setup of entry."""
     await setup_axis_integration(hass)
-    assert len(hass.data[axis.DOMAIN]) == 1
-    assert MAC in hass.data[axis.DOMAIN]
+    assert len(hass.data[AXIS_DOMAIN]) == 1
+    assert MAC in hass.data[AXIS_DOMAIN]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)